### DISCUSSION TITLE
how to enable autoplay ?

### DISCUSSION THREAD

**Discussion by ank-007** [CefView/QCefView#328] [Q&A] [answered]
@leashi @OgreTransporter @tishion @le0zh @ZhouYing-bit .if i pass a youtube url with autoplay=1 it should start playing the video. I am looking for something like PlaybackRequiresUserGesture which we set to false in QWebEngineSetting to make autoplay to happen.

**Comment by be-water-myFriend** [comment, marked as answer]
@ank-007  You could give this setup a try.
  
  QCefConfig config;
  config.addCommandLineSwitchWithValue("autoplay-policy", "no-user-gesture-required");

**Comment by ank-007** [comment]
thank you so much, it worked.
